addons/hw_posbox_homepage/controllers/main.py

```diff
@@ -90,13 +90,15 @@ def get_homepage_data(self):
 
         pos_device = self.get_pos_device_status()
         iot_device = []
-        for status in pos_device:
-            if pos_device[status]['status'] == 'connected':
-                iot_device.append({
-                    'name': status,
-                    'type': 'device',
-                    'message': ' '.join(pos_device[status]['messages'])
-                })
+
+        if not iot_devices:
```

Comment:
why this change? why in this commit?

Reply:
As a consequence of the changes made for backward compatibility a 'scale' driver appears in the IoT box homepage under device. This is the way to avoid it. It should go on the next IoT box image. Should I put that in the different PR?